api/doc/openapi.yaml

```diff
@@ -442,6 +442,8 @@ paths:
           schema:
             type: string
           example: "sent_events"
+        - $ref: '#/components/parameters/StartEvent'
+        - $ref: '#/components/parameters/LimitEvent'
```

Comment:
`EventsStart` and `EventsLimit` sounds slightly more natural to me.. I'm not sure though..

Reply:
updated the names